src/NServiceBus.Core/Recoverability/SecondLevelRetries/CustomSecondLevelRetryPolicy.cs

```diff
@@ -14,7 +14,7 @@ public override bool TryGetDelay(IncomingMessage message, Exception ex, int curr
         {
             delay = customRetryPolicy(message);
 
-            return delay != TimeSpan.Zero;
+            return delay != TimeSpan.MinValue;
```

Comment:
question to the task-force: the retries docs seems to correctly document MinValue already. Did you check for other usages (e.g. samples?)